image/templates/helm/shared/templates/02-scanner-07-service.yaml

```diff
@@ -19,6 +19,11 @@ spec:
     - name: grpcs-scanner
       port: 8443
       targetPort: 8443
+    {{ if ._rox.scanner.exposeMonitoring -}}
+    - name: monitoring
+      port: 9090
+      targetPort: monitoring
```

Comment:
feel free to ignore, but should we use a concrete number like the rest (or switch the others to named)?

Reply:
I used the `monitoring` name to be consistent with how monitoring is configured for Central. Having a consistent name is also useful when configuring the Prometheus pod monitors for both Central and Scanner. Perhaps we should add names for the other ports as well, but I don't think it would be in scope here.